### PR TITLE
Add mako

### PR DESCRIPTION
**Pull request by Korijn**
Required for #960

### CODE REVIEW

**Comment by ocefpaf** [comment]
This LGTM. Just waiting for AppVeyor.

**Comment by Korijn** [comment]
Sweet! :hand:

**Comment by ocefpaf** [comment]
:hand:

File: recipes/mako/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+source:
+  fn: Mako-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/M/Mako/Mako-{{ version }}.tar.gz
+  sha256: fed99dbe4d0ddb27a33ee4910d8708aca9ef1fe854e668387a9ab9a90cbf9059
+
+build:
+  preserve_egg_dir: True
+  entry_points:
+    - mako-render=mako.cmd:cmdline
+  script: python setup.py install
```
Comment (jakirkham) [inline recipes/mako/meta.yaml]: Please add these arguments in the feedstock `--single-version-externally-managed --record=record.txt`.

Comment (Korijn) [inline recipes/mako/meta.yaml]: I will! What do they do? :)

Comment (jakirkham) [inline recipes/mako/meta.yaml]: Basically, they tell `setuptools` to not manage this project directly as it is being managed by another package manager (in our case `conda` 😉). See this [info]( http://setuptools.readthedocs.io/en/latest/setuptools.html?highlight=single-version-externally-managed#install-command ).

**Comment by jakirkham** [comment]
@zzzeek, just letting you know we have added `mako` to [conda-forge]( http://conda-forge.github.io/ ). This will allow users to easily install this with [`conda`]( http://conda.pydata.org/docs/ ) (`conda install -c conda-forge mako`). The maintenance burden here is light as all testing releases are built and deployed automatically with CI. Changes to how it is being built can be controlled by changing the recipe in [this repo]( https://github.com/conda-forge/mako-feedstock ). If you have any interest in being added as a maintainer (to make new releases for instance), we would love to have you on board. Thanks for your work on `mako`. 😄

**Pull request merged**